default/scripting/policies/RACIAL_PURITY.focs.txt

```diff
@@ -9,25 +9,68 @@ Policy
     effectsgroups = [
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
         
+        // This policy disables influence and research foci for non-capital, non-exobot species
+        // See /species/common/focus.macros
+
+        // Creates a special to ensure that deadopting this policy has bad consequencies for a while.
+        EffectsGroup
+            scope = Source
+            activation = Not HasSpecial name = "RACIAL_PURITY_MASTER_SPECIAL"
+            effects = AddSpecial name = "RACIAL_PURITY_SLAVE_SPECIAL"
```

Comment:
here is your bug: this should add MASTER, not slave. Somewhere you have to add the master special.

Reply:
No, I found this bug two days ago and solved in my local. It doesn't solve the problem. See my last comment in this thread, https://github.com/freeorion/freeorion/pull/4116#issuecomment-1237730414, or the the discussion thread, https://www.freeorion.org/forum/viewtopic.php?p=114017#p114017

Reply:
Ok, I can only comment on the published code.

Reply:
I'm trying to update the PR with the new strategy, using the special capacity, and it is giving me problems. But as soon as I get it I'll update the PR.